pkg/commonconfig/commonconfig.go

```diff
@@ -21,7 +21,8 @@ const (
 )
 
 type CommonConfig struct {
-	ConfigFlags *genericclioptions.RESTClientGetter
+	ConfigFlags  *genericclioptions.RESTClientGetter
+	installation *installation.Installation
```

Comment:
Storing Installation in the common config, so that it can be loaded from Athena only once and re-used multiple times in different places of kgs.